Rubberduck.Refactorings/ExtractInterface/ExtractInterfaceRefactoring.cs

```diff
@@ -129,17 +130,48 @@ private void AddInterfaceClass(Declaration implementingClass, string interfaceNa
                         var optionPresent = interfaceModule.CountOfLines > 1;
                         if (!optionPresent)
                         {
-                            interfaceModule.InsertLines(1, $"{Tokens.Option} {Tokens.Explicit}{Environment.NewLine}");
+                            interfaceModule.InsertLines(1, "Option Explicit" + Environment.NewLine);
+                        }
+
+                        interfaceModule.InsertLines(1, "'@Interface");
+                        interfaceModule.InsertLines(4, interfaceBody);
+
+                        if (interfaceInstancing == ClassInstancing.Public)
+                        {
+                            AddExposedAttribute(components, interfaceComponent);
                         }
-                        interfaceModule.InsertLines(3, interfaceBody);
                     }
                 }
             }
         }
 
+        private void AddExposedAttribute(IVBComponents components, IVBComponent interfaceComponent)
+        {
+            try
+            {
+                var tempExportDirectory = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + System.IO.Path.DirectorySeparatorChar + "Rubberduck" + System.IO.Path.DirectorySeparatorChar;
+                var tempFile = interfaceComponent.ExportAsSourceFile(tempExportDirectory + interfaceComponent.Name);
```

Comment:
could be replaced with `interfaceComponent.ExportToTempFile();`, removing the `tempExportDirectory` creation.